com.unity.shadergraph/Documentation~/Parallax-Mapping.md

```diff
@@ -0,0 +1,13 @@
+# Parallax Mapping Node
+
+The Parallax Mapping Node allows you to create a parallax effect that displaces a Material's UVs to create the illusion of depth inside a Material. Uses single step process that does not account for occlusion.
```

Comment:
My suggestion:
The Parallax Mapping Node lets you create a parallax effect that displaces a Material's UVs to create the illusion of depth inside a Material. This implementation uses the single step process that does not account for occlusion. For information on how the effect looks, see the [Height Map](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterHeightMap.html) page.